dist/templates/ovnkube-master.yaml.j2

```diff
@@ -198,6 +198,9 @@ spec:
           requests:
             cpu: 100m
             memory: 300Mi
+          limits:
+            cpu: 1
```

Comment:
Do we have any idea what the performance impact will be, if any? eg, does this prevent using multiple OS threads on multiple CPUs, or just limit the total amount of CPU usage to what one CPU would provide?  On the node side we don't care because we aren't CPU-bound there at all.